templates/municipal_manager/cities.html.twig

```diff
@@ -0,0 +1,64 @@
+{% extends 'municipal_manager/_layout.html.twig' %}
+
+{% import 'helper/vote_result_macro.html.twig' as _helper %}
+
+{% block municipal_manager_content %}
+    <section>
+        <table class="datagrid__table-manager">
+            <thead>
+            <tr>
+                <td class="space--0-15">Ville</td>
+                <td class="text--center">Dernière<br/>modification</td>
+                <td>Action</td>
+            </tr>
+            </thead>
+            <tbody>
+            {% for city in app.user.municipalManagerRole.cities %}
+                <tr>
+                    <td>
+                        <div class="l__row l__row--h-start">
+                            <div class="l__col">
+                                <strong>{{ city.name }} ({{ city.inseeCode }})</strong>
+                                <br />
+                                <span class="text--small">
+                                    {{ city.postalCodes|join('/') }}, {{ city.country|countryName }}
```

Comment:
instead of the country (that will ALWAYS be France) , you can display the department instead `{{ city.department }}`

edit: nope, you can do that after my [PR](https://github.com/EnMarche/en-marche.fr/pull/4295) is merged